main.go

```diff
@@ -171,9 +171,13 @@ func main() {
 	}
 
 	certsReady := make(chan struct{})
-	if err = cert.CertsManager(mgr, cfg, certsReady); err != nil {
-		setupLog.Error(err, "unable to setup cert rotation")
-		os.Exit(1)
+	if cfg.InternalCertManagement != nil && *cfg.InternalCertManagement.Enable {
```

Comment:
[Default value of this field](https://github.com/kubernetes-sigs/jobset/blob/ab3881469b6c5fbd5e4a8e979bf041013eb1add0/api/config/v1alpha1/defaults.go#L71) is to true, so this should be backwards compatible?